frontend/src/redux/auth/actions.ts

```diff
@@ -0,0 +1,34 @@
+import { Dispatch } from "redux"
+
+import { Auth } from "../../types"
+import { UpdateAuthAction, UPDATE_AUTH } from "./action-types"
+
+
+export function updateAuth(auth: Auth): UpdateAuthAction {
+    return {
```

Comment:
You've got inconsistent indentation in some places. Eg, here it's 4 spaces instead of 2. Is the linter set up to catch these, and is it running in Travis?

Reply:
Odd; airbnb doesn't specify a rule around spaces. guess it's too 🔥 for them to make a decision. #spacesforever